cmd/jaeger/internal/integration/e2e_integration.go

```diff
@@ -77,9 +94,23 @@ func (s *E2EStorageIntegration) e2eInitialize(t *testing.T, storage string) {
 	t.Log("Jaeger-v2 is ready")
 	t.Cleanup(func() {
 		require.NoError(t, cmd.Process.Kill())
+		if t.Failed() {
+			// A Github Actions special annotation to create a foldable section
+			// in the Github runner output.
+			// https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#grouping-log-lines
+			fmt.Println("::group::Jaeger-v2 binary logs")
```

Comment:
I don't think it'll work with print, it's not guaranteed to be properly synced with t.Logf

Reply:
What if we use fmt.Printf to dump the binary log files? It will output the logs like this
```
::group::Jaeger-v2 binary logs
Jaeger-v2 output logs:
Jaeger-v2 error logs:
2024/05/08 13:35:55 application version: git-commit=, git-version=, build-date=
2024-05-08T13:35:55.403+0700	info	service@v0.100.0/service.go:102	Setting up own telemetry...
...
```

This is how it looks in real case https://github.com/jaegertracing/jaeger/actions/runs/8997482037/job/24715702727?pr=5431#step:9:874.